DEVELOPMENT.md

```diff
@@ -57,18 +57,23 @@ make test
 
 ### Integration tests
 
-On a GCP project where Kritis has already been installed, this will prepare a new cluster named `kritis-integration-test`:
+On a GCP project where Kritis has already been installed (follow [instructions](docs/install.md)), this will prepare a new cluster named `kritis-integration-test`:
 
 ```shell
-make -e GCP_PROJECT=<project id> setup-integration-local
+make -e GCP_PROJECT=${PROJECT_ID} setup-integration-local
 ```
 
+NOTE: the `PROJECT_ID` here is from the installation instructions environment
+variable.
+
 As you develop, you can build new test images and run the integration test on demand:
 
 ```shell
-make -e GCP_PROJECT=<project id> integration-local
+make -e GCP_PROJECT=${PROJECT_ID} integration-local
 ```
 
+NOTE: you'll be charged for using GCP resource to run integration tests. See [Free Tier](https://cloud.google.com/free) for information on how to try out GCP. Alternatively, you can choose to have the integration tests run as part of the PR instead. Fixes to the testing infrastructure to address this problem are WIP.
```

Comment:
Is there an open issue on Github that could be linked to?

Reply:
There isn't, because it's all internal work that the community is unlikely to be able to help with due to the need to streamline Grafeas and Kritis. I'm tracking it internally though.